src/parsers/pairs.ts

```diff
@@ -1,10 +1,5 @@
-import type { CSSProperties } from 'react';
+import type { CSSProperties, RefObject } from 'react';
```

Comment:
You can remove the `RefObject` import here, this is why the CI run is failing 😄 

```suggestion
import type { CSSProperties } from 'react';
```